app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,214 @@
+    <string name="guide_privacy_item2">Your identity is kept secret with the help of random identification codes.</string>
+    <string name="guide_privacy_item3">The app does not collect location data.</string>
+
+    <string name="guide_button_back">Back</string>
+
+    <string name="disable_title">COVID-19-\u200Bexposure\u200Bnotification enabled</string>
+    <string name="disable_subtitle">Do you need to turn off exposure notifications?</string>
+    <string name="disable_body">In some situations, it may be necessary to temporarily disable exposure notifications. This may be the case, for example, if you are a healthcare professional and do not want to use the application in your workplace.\n\nThe application will not work while the exposure notifications are disabled.</string>
+    <string name="disable_note">To take care of your health and the health of your loved ones, it is recommended that you turn off exposure notifications only when absolutely necessary. Remember to re-enable them as soon as possible.</string>
+    <string name="disable_turn_off_note">Disable the COVID-19 exposure log and notifications</string>
+    <string name="disable_button_turn_off">Disable</string>
+
+    <string name="disable_confirm_title">Are you sure you want to disable COVID-19 exposure notifications?</string>
+    <string name="disable_confirm_message">If you disable the exposure notifications, Corona Alert will not notify you of any exposure to the coronavirus.</string>
+
+    <string name="enable_title">The COVID-19-\u200Bexposure\u200Bnotifications are disabled</string>
+
+    <string name="enable_body">Exposure notifications are disabled, Corona Alert does not follow phones near you.</string>
+    <string name="enable_alert">To ensure your own and your loved ones' health, it is advisable to switch on exposure notifications as soon as possible.</string>
+    <string name="enable_note">Enable COVID-19 exposure notifications</string>
+
+    <string name="enable_err_title">The exposure log could not be initialized</string>
+    <string name="enable_err_missing_capability">Your device is not supported</string>
+    <string name="enable_err_generic">Unknown error</string>
+    <string name="enable_err_dismiss">Dismiss</string>
+    <string name="enable_error_api_code">An unexpected error has occurred, code:% d</string>
+
+    <string name="onboarding_done">Onboarding complete!</string>
+
+    <string name="intro_header_title">Thank you for downloading Corona Alert</string>
+    <string name="intro_header_body">Corona Alert is an application (app) for which the Institute for Health and Welfare is responsible. The app strives to prevent the spread of the coronavirus. The app effectively guards your privacy.</string>
```

Comment:
```suggestion
    <string name="intro_header_body">Corona Alert is an application (app) for which the Finnish Institute for Health and Welfare is responsible. The app strives to prevent the spread of the coronavirus. The app effectively guards your privacy.</string>
```